assembly/src/main/resources/features.xml

```diff
@@ -16,6 +16,7 @@
 <features name="karaf-cellar-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.2.0 http://karaf.apache.org/xmlns/features/v1.2.0">
 
     <repository>mvn:org.apache.jclouds.karaf/jclouds-karaf/${jclouds.version}/xml/features</repository>
+    <repository>mvn:org.apache.cxf.karaf/apache-cxf/3.0.2/xml/features</repository>
```

Comment:
As I said in the upper comment, I'm expecting fabric8 guys to provide a feature for ```io.fabric8-kubernetes-api``` that will fix this.